src/controllers/auth.controller.ts

```diff
@@ -49,8 +50,8 @@ export class AuthController {
 
     // user matches email + password, create a token
     const token = await createJWT(user);
-    return {
-      token
-    };
+    response.set('access-token', token);
```

Comment:
This will return token in the header, we don't want that. Token should be part of the response body

Reply:
<img width="1386" alt="Screen Shot 2020-12-01 at 13 38 25" src="https://user-images.githubusercontent.com/3788239/100769255-8b6d1100-33da-11eb-8392-8ab34bd3c962.png">

Reply:
Are you sure about the token in body? The current react base reads it from headers.